### PR TITLE
fixed #207 Print the subset sum to K

### LINKED ISSUE #207: Print the subset sum to K (Recursive Problem)
I want't to code this problem in which a array is given and an integer is also given.Want to print the output of the subsets of array which sums to k.

### PR DESCRIPTION
**Pull request by Anujg935**
implemented the solution to problem.

### CODE REVIEW

**Comment by Anujg935** [comment]
please have a look at code and merge it .

**Comment by Anujg935** [comment]
have a look at code now

File: Algorithms/Recursion/Print_the_subset_sum_to_K.cpp [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	cin>>number_of_elements;
+	//input array
+	int *arr = new int[number_of_elements];
+	cout<<"Enter "<<number_of_elements<<" Elements"<<endl;
+	for(int i=0;i<number_of_elements;i++){
+		cin>>arr[i];
+	}
+	//calling ths function with 3 parameters 
+	//1st parameter is array 2nd is the size of array and 3rd is the sum for which the subsets are to be printed
+	printSubsetSumToK(arr, 9, 6);
```
Comment (aditya81070) [inline Algorithms/Recursion/Print_the_subset_sum_to_K.cpp]: You should also prompt user to input the value of k(3rd argument)

**Comment by Anujg935** [comment]
Can you tell me on which part of the program is spelling mistakes

File: Algorithms/Recursion/Print_the_subset_sum_to_K.cpp [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+//printSubsets() is a helper function 
+//this function breaks the input array in two parts 
+//i.e including the 1st element of array in the sum and excluding it from the sum
+void printSubsets(int input[],int n, int output[],int m,int k){
+	//base case for recursive function
+	//checks if the size of infput array is empty or not
+	if(n==0){
+		//checks if the sum is reduced to zero or not
+    	if(k==0){
```
Comment (vikasgola) [inline Algorithms/Recursion/Print_the_subset_sum_to_K.cpp]: Please correct your indentation.

**Comment by vikasgola** [comment]
@Anujg935 there are many indentation errors correct them.

**Comment by Anujg935** [comment]
i am not able to find much indentation errors but fixe some which i am able to saw.If you find some other than please let me know.

**Comment by Anujg935** [comment]
now have a look if everything is okay

**Pull request merged**